website/blog/2023-03-10-databend-weekly-84.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "This Week in Databend #84"
+date: 2023-03-10
+slug: 2023-03-10-databend-weekly
+tags: [databend, weekly]
+description: "Learn about the latest updates of Databend in a week!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Carlosfengv
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: drmingdrmer
+  - name: dusx1981
+  - name: everpcpc
+  - name: jun0315
+  - name: leiysky
+  - name: mergify[bot]
+  - name: PsiACE
+  - name: quaxquax
+  - name: RinChanNOWWW
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: xinlifoobar
+  - name: Xuanwo
+  - name: xudong963
+  - name: youngsofun
+  - name: zhang2014
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+> :loudspeaker: This week we announced the official release of Databend v1.0. This is an important milestone, to learn more, please check out our blog: https://databend.rs/blog/databend-release-v1.0
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### SQL: `REPLACE INTO`
+
+The `REPLACE INTO` statement is a powerful way to insert or update data in Databend. It allows you to specify a conflict key (not necessarily a primary key) that determines whether a new row should be inserted or an existing row should be updated.
+
+If a row with the same conflict key already exists in the table, it will be replaced by the new data. Otherwise, a new row will be added with the new data. You can use this statement to easily sync data from different sources or handle duplicate records.
+
+```sql
+#> CREATE TABLE employees(id INT, name VARCHAR, salary INT);
+#> REPLACE INTO employees (id, name, salary) ON (id) VALUES (1, 'John Doe', 50000);
+#> SELECT  * FROM Employees;
++------+----------+--------+
+| id   | name     | salary |
++------+----------+--------+
+|    1 | John Doe |  50000 |
++------+----------+--------+
+```
+
+If you want to learn more details about `REPLACE INTO` statement, please read the materials listed below.
+
+- [Docs | DML Commands - REPLACE](https://databend.rs/doc/sql-commands/dml/dml-replace)
+- [PR | feat: replace into statement](https://github.com/datafuselabs/databend/pull/10191)
+
+Also, we plan to gradually transition to the `MERGE INTO` statement based on this, and if you are interested in this topic, you can also follow this issue:
+
+- [Issue | tracking: `MERGE` #10373](https://github.com/datafuselabs/databend/issues/10373)
+
+### RFC: Transform During Load
+
+Currently, we support stage table function, so we can transform when insert from stage into table.
```

Comment:
Databend is currently capable of transforming and inserting data from a stage into a table. For example, you can run a SQL statement like this:

```sql
insert into table1 from (select c1， c2 from @stage1/path/to/dir);
```
The COPY INTO command needs a similar feature as well to support incremental data loading from a stage. 

If you're interested, check the following RFCs: